components/tabs/tab.js

```diff
@@ -32,21 +32,30 @@ class Tab extends TabMixin(LitElement) {
 				box-shadow: 0 0 0 2px var(--d2l-color-celestine);
 				color: var(--d2l-color-celestine);
 			}
+			.d2l-tab-text-skeletize-override {
+				min-width: 50px;
+			}
+			:host([skeleton]) .d2l-tab-text.d2l-skeletize::before {
+				bottom: 0.15rem;
```

Comment:
I have no idea how we would want tabs to actually look in a vertical writing mode, but we might now lean into `inset-block-start`/`inset-block-end`.